epoxy-sample/src/main/java/com/airbnb/epoxy/sample/models/ColorModel.java

```diff
@@ -28,6 +28,16 @@
   @EpoxyAttribute boolean playAnimation;
   @EpoxyAttribute(DoNotHash) View.OnClickListener clickListener;
 
+  @EpoxyAttribute private boolean test;
```

Comment:
Did you mean to include this?

If we want a test we can put it in the integration test module, since otherwise this sample model might be changed later if we forget it is intended as a test